lib/ome/files/MetadataTools.cpp

```diff
@@ -548,7 +548,9 @@ namespace ome
           try
             {
               OMEXMLMetadata& omexml(dynamic_cast<OMEXMLMetadata&>(store));
-              addMetadataOnly(omexml, s);
+              if (omexml.getTiffDataCount(s) == 0 ||
```

Comment:
Should this not be an `&&` which would explain the error in https://github.com/ome/ome-files-cpp/pull/26#issuecomment-235922852?